### PR TITLE
Add support for 5060830P7

### PR DESCRIPTION
**Pull request by SHxKM**
Couldn't find where I can add a picture of this product.

### CODE REVIEW

File: devices/philips.js [hunk 1]
```diff
@@ -1221,6 +1221,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['5060830P7_01', '5060830P7_02', '5060830P7_03', '5060830P7_04'],
+        model: '5060830P7',
+        vendor: 'Philips',
+        description: 'Hue White & Color ambience Centris ceiling light (3 spots)',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```
Comment (Koenkk) [inline devices/philips.js:1230]: Please add the color temp range: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature (also for the other 2 prs)

Comment (SHxKM) [reply devices/philips.js:1230]: Ah, missed this one. Will do once I get back on site.

Comment (SHxKM) [reply devices/philips.js]: @Koenkk done for this, and all others I've submitted.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**